utility/src/main/java/org/oppia/android/util/parser/ImageLoader.kt

```diff
@@ -23,11 +31,21 @@ interface ImageLoader {
   /**
    * Loads a vector drawable at the specified [imageUrl] into the specified [target]. Note that this
    * is an asynchronous operation, and may take a while if the image needs to be downloaded from the
-   * internet.
+   * internet. Optional [transformations] may be applied to the image.
    */
   fun loadSvg(
     imageUrl: String,
     target: ImageTarget<PictureDrawable>,
     transformations: List<ImageTransformation> = listOf()
   )
+
+  /**
+   * Loads a drawable resource at the specified [imageDrawable] into the specified [target].
+   * Optional [transformations] may be applied to the image.
+   */
```

Comment:
```suggestion
  /**
   * Loads the specified [imageDrawable] resource into the specified [target]. Optional [transformations] may be applied to the image.
   */
```

Suggested rewording to clarify since the previous wording makes the image drawable sound a bit like a URL. (New version needs minor line wrapping).

Reply:
Done.